packages/terra-clinical-label-value-view/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "terra-clinical-label-value-view",
+  "main": "lib/LabelValueView.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "terra-clinical-label-value-view",
```

Comment:
A description should be added here, most people copy paste the description from the readme